### DISCUSSION TITLE
Mark all files as viewed / not viewed

### DISCUSSION THREAD

**Discussion by eduherminio** [community/community#7881] [Pull Requests]
It would be awesome to be able to mark all files in a PR as viewed/not viewed at the same time, in the same as `alt+click `collapses/extends them all.

[Migrated from https://github.com/isaacs/github/issues/1748]

**Comment by Febbe** [comment]
I think this is only helpful in combination with a tree-file-view.

**Comment by jaredellison-nyt** [comment]
Currently I do this using a javascript [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) saved as "uncheck" so I can type "uncheck" as a command in my url bar:
```js
javascript: document.querySelectorAll('.js-reviewed-checkbox').forEach(input => !input.checked || input.click());
```
I would love to see this feature added to the UI!

**Comment by ml242** [comment]
Would find this useful, please have my upvote.

**Comment by gabelepoudre** [comment]
Seconded, upvoted. This would be especially valuable to me if a button appeared by context at the end of all diffs. I dislike being half-way through a large review and realizing I need to go back through my changes and hit "viewed"
